internal/json.go

```diff
@@ -112,6 +114,7 @@ func RenderJSONView(tasks []Task) error {
 			ID:            info.configDump.ID,
 			Runtime:       info.configDump.OCIRuntime,
 			Created:       info.containerInfo.Created,
+			Checkpointed:  info.configDump.CheckpointedAt.Format(time.RFC3339),
```

Comment:
It looks like Podman currently does not set `CheckpointedAt`

Reply:
Interesting. We should set that. I will change this PR to skip it if empty.

Reply:
Removed the zero value if not set as in the Podman case.